JapaneseFilter/sections/specific.txt

```diff
@@ -2299,8 +2299,12 @@ openloadpro.com,javfan.one,javtheater.com,jav.direct,javmix.tv##.__isboostOverCo
 9tsu.*,jshow.tv##.ads-system
 9tsu.*,jshow.tv##.cactus-sidebar-content > .widget_text
 ||rakuten.co.jp/com/img/thumb/logout/bnr/
+toyokeizai.net###adexchange
+toyokeizai.net###induction-banner
```

Comment:
Please move it to the `AdGuard Annoyances - self-promo`.

Reply:
ok.done